src/components/core/DpEditor/DpLinkModal.vue

```diff
@@ -46,7 +46,7 @@
 </template>
 
 <script>
-import DpButtonRow from '../DpButtonRow'
+import DpButtonRow from '../../DpButtonRow/DpButtonRow'
```

Comment:
The longer i look at those paths, the more i want to [use aliases to access other modules](https://webpack.js.org/configuration/resolve/#resolvealias). However i guess this can be a completely separate PR.

Reply:
I was wondering about the paths, too. We could also import from the index file, would you consider that a better solution?